linear/linear.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program solves a linear inequality Ax + B > 0.")
+	fmt.Println("Enter A and B:")
+	var a, b float64
+	fmt.Scanln(&a, &b)
+	result := -b / a
```

Comment:
Division by 0 is plausible when `result` is set here. Please ensure 0 division is eliminated.